soc/v2/nios2-qemu/Kconfig.soc

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_NIOS2_QEMU
+	bool
+
+config SOC
+	default "nios2-qemu" if SOC_NIOS2_QEMU
```

Comment:
open question.

The board is named `qemu_nios2`, the SoC `nios2-qemu`, should we swap the SoC name so that `qemu` is first, similar to the board name order ?

Reply:
Swapped